src/poker/game/PokerUtils.java

```diff
@@ -16,10 +16,10 @@
 //zapytanie czy chcesz doładować stacka jeśli masz mniej niż 100bb
 //ładne printy jaki jest board/ kto wygrał z jakim układem/print equity na all in w trakcie gry/
 //czyszenie po preflopie,flopie,turnie,river
-public class MakeBoardAndCards {
+public class PokerUtils  {
     Deck deck;
-    public MakeBoardAndCards(Deck deck){
-        this.deck = deck;
+    PokerUtils(Deck deck){
+        this.deck = new Deck();
     }
 
     public HandsAndBoard makeHandsBoard(){
```

Comment:
ogolnie wynieslismy to do klasy Deck wiec ta klasa nie ma nic czego uzywamy wiec do wywalenia to jest. Ale miej z tylu glowy ze taka klasa jak PokerUtils moze ci sie przydac bo mozna do niej wkladac funkcje pomocnicze jak bedzie potrzeba